extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -1250,6 +1263,10 @@ export class AmpLightboxGallery extends AMP.BaseElement {
   closeGallery_() {
     return this.mutateElement(() => {
       this.container_.removeAttribute('gallery-view');
+      Services.ownersForDoc(this.element)./*OK*/ scheduleUnlayout(
```

Comment:
nit: This is done 4x in this class, make it a helper

Reply:
Agreed, but would prefer in a different PR. 

There are lots of examples of where this extension is breaking rules and I am fearful of making significant changes without:
1. Better tests
2. More consideration of what the implementation should own.